server/list.go

```diff
@@ -134,26 +135,23 @@ func (l *listManager) GetIssueList(userID, listID string) ([]*ExtendedIssue, err
 	return extendedIssues, nil
 }
 
-func (l *listManager) CompleteIssue(userID, issueID string) (*ExtendedIssue, error) {
+func (l *listManager) CompleteIssue(userID, issueID string) (issue *Issue, foreignID string, err error) {
```

Comment:
0/5 why name the return parameters? Seems unnecessary here.

Reply:
Just because of parallelism with the other similar functions.